contrib/dockerd-rootless-setuptool.sh

```diff
@@ -78,6 +79,10 @@ init() {
 		exit 1
 	fi
 
+	# Set USERNAME from `id -un` and potentially protect backslash
+	# for windbind/samba domain users
+	USERNAME=$(id -un | sed 's/\\/\\\\/g')
```

Comment:
I feel this variable should be named like `USERNAME_ESCAPED`

Reply:
Good idea.
Just implemented it.